### DISCUSSION TITLE
format js code in fastn with prettier

### DISCUSSION THREAD

**Discussion by siddhantk232** [fastn-stack/fastn#1649] [Ideas & RFCs]
suggested cmd to format in CI, git hooks and, manual: `prettier --write --tab-width 4 fastn-js/js/ftd.js`

reference:

- https://github.com/fastn-stack/fastn/pull/1645#issuecomment-1870971602

**Comment by amitu** [comment]
@harshdoesdev has created a [PR for this](https://github.com/fastn-stack/fastn/pull/1654).
